js/customize-snapshots.js

```diff
@@ -835,6 +837,149 @@
+					_.each( control.settings, function( setting ) {
+						setting.unbind( component.onConflictFirstChange );
+					} );
+					control.notifications.remove( component.conflict.notificationCode );
+					component.addConflictButton( control );
+				} );
+			}
+		} );
+	};
+
+	/**
+	 * Add conflict button icon on first change and trigger handleConflictRequest.
+	 *
+	 * @param {object} control where conflict button will be added.
+	 *
+	 * @return {void}
+	 */
+	component.addConflictButton = function addConflictButton( control ) {
+		control.deferred.embedded.done( function() {
+			var onFirstChange, hasDirty;
+
+			if ( ! control.setting ) {
+				return;
+			}
+
+			onFirstChange = component.onConflictFirstChange = function() {
+				_.each( control.settings, function( setting ) {
+					setting.unbind( onFirstChange );
+					component.handleConflictRequest( setting );
+				} );
+			};
+
+			hasDirty = _.find( control.settings, function( setting ) {
+				return setting._dirty;
+			} );
+			if ( hasDirty ) {
+				onFirstChange();
+			} else {
+				_.each( control.settings, function( setting ) {
+					setting.bind( onFirstChange );
+				} );
+			}
+		} );
+	};
+
+	/**
+	 * Handles the snapshot conflict request
+	 *
+	 * @param {object} setting to check conflicts
+	 *
+	 * @return {void}
+	 */
+	component.handleConflictRequest = function conflictRequest( setting ) {
+		if ( component.conflict._currentRequest ) {
+			component.conflict._currentRequest.abort();
+			component.conflict._currentRequest = null;
+		}
+		if ( component.conflict._debouncedTimeoutId ) {
+			clearTimeout( component.conflict._debouncedTimeoutId );
+			component.conflict._debouncedTimeoutId = null;
+		}
+		component.conflict.pendingRequest[setting.id] = setting.findControls();
+
+		component.conflict._debouncedTimeoutId = setTimeout(
+			function sendConflictRequest() {
+				var data, settingIds;
+				settingIds = _.keys( component.conflict.pendingRequest );
+				if ( _.isEmpty( settingIds ) ) {
+					return;
+				}
+				data = {
+					setting_ids: settingIds,
+					nonce: api.settings.nonce.snapshot,
+					customize_snapshot_uuid: component.data.uuid
+				};
+				component.conflict._currentRequest = wp.ajax.post( 'customize_snapshot_conflict_check', data );
+
+				component.conflict._currentRequest.done( function( returnData ) {
+					var multiple, controls, buttonTemplate, notification;
+					if ( ! _.isEmpty( returnData ) ) {
+						_.each( returnData, function( value, key ) {
+							component.conflict.controls[key] = $( $.trim( component.conflict.warningTemplate( {
+								setting_id: key,
+								conflicts: value
+							} ) ) );
+							multiple = false;
+							controls = component.conflict.pendingRequest[key];
+							buttonTemplate = $( $.trim( component.conflict.icon( {
+								setting_id: key
+							} ) ) );
+							_.each( controls, function( control ) {
+								control.notificationsTemplate = component.conflict.notificationTemplate;
```

Comment:
Yes we are overriding `control.notificationsTemplate` so it allows markup as a notification message.